src/types.ts

```diff
@@ -112,3 +112,20 @@ type TreeDiffModified = {
 };
 export type TreeDiffEntry = TreeDiffAdded | TreeDiffRemoved | TreeDiffModified;
 export type TreeDiff = TreeDiffEntry[];
+
+export type CruftPackMode =
+    /** Processes all packs */
+    | 'include'
```

Comment:
You didn't accept my suggestions to add `true` / `false` aliases for `"include"` / `"exclude"`, or missed it?